README.md

```diff
@@ -7,9 +7,10 @@ Represent a minecraft block with its associated data
 ## Usage
 
 ```js
-const Block = require('prismarine-block')('1.8')
+const registry = require('prismarine-registry')('1.8')
```

Comment:
Can we support both version and registry and have example and test for both ?
It would keep things simple and compatible

Reply:
Yes, this keeps the old string parameter around for compatibility. We could add alternative examples beyond the API for passing the registry/string, but that would also need to happen in all of the prismarine- packages also. Not sure it's necessary for one extra call, I was going to add both but it seemed confusing without adding extra context to which one is appropriate